ydb-global.h

```diff
@@ -31,10 +56,13 @@ class c_ydb_global {
 	operator string();
 	string next();
 	string previous();
+	// REVIEW: this should be documented. 1 means YDB_DEL_TREE, 0 means YDB_DEL_NODE. I would even go so far as making it an `enum`.
 	void kill(int = 1);
+	// REVIEW: the `int` here is confusing, but is consistent wtih `ydb_data`. Consider using an enum instead, but keeping as an int seems fine if you think consistency is better.
 	unsigned int  data();
 	int rc();
 	c_ydb_global operator = (c_ydb_global &);
+// REVIEW: why use `protected` instead of `private`? Why add this to the public header instead of the .cpp file?
```

Comment:
When someone inherits the class (public inheritance):
public stays public, protected staysprotected, private will not be inherited.
In the class itself "protected" has the same behaviour as "private"